src/scene/camera.js

```diff
@@ -203,313 +478,26 @@ Object.assign(Camera.prototype, {
         return worldCoord;
     },
 
-    /**
-     * @private
-     * @function
-     * @name pc.Camera#getClearOptions
-     * @description Retrieves the options used to determine how the camera's render target will be cleared.
-     * @returns {object} The options determining the behaviour of render target clears.
-     */
-    getClearOptions: function () {
-        return this._clearOptions;
-    },
-
     _evaluateProjectionMatrix: function () {

```

Comment:
wondering why this isn't just wrapped into projectionMatrix getter?

Reply:
It does need tidying up. But this PR is really, mainly about adopting the new component format and nicely dividing component functionality (property management) from camera functionality (pc.Camera). Let's address this in a follow up PR.